draft-ietf-capport-architecture.xml

```diff
@@ -861,10 +861,27 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
      <?rfc include="reference.I-D.ietf-intarea-provisioning-domains.xml"?>
    </references>
 
-<!--
-   <section anchor="app-additional" title="Additional Stuff">
-     <t>This becomes an Appendix.</t>
+   <section anchor="app-additional" title="Existing captive portal detection implementations">
+     <t>
+       Operating systems and user applications may perform various tests when
+       network connectivity is established to determine if the device is
+       attached to a network with a captive portal present. A common method is
+       attempt to make a HTTP request to a known, vendor hosted endpoint with an
+       expected response. This check should not be secured with TLS, as a network
+       with a captive portal may intercept the connection, leading to a host
+       name mismatch.
```

Comment:
This may be implied, but we may want to describe what indicates to the device that it is behind a captive portal. In this case, the fact that the response is not the known response.

I'm not opposed to leaving out the criteria for a positive result, but I do prefer to be explicit.

Reply:
To address this, I think I would have to leave `"...with an expected response."` and explain that unexpected responses would flag the network as having a captive portal. @martinthomson does this make more sense, given your [comment](https://github.com/capport-wg/architecture/pull/26#discussion_r270100694) or is there a better way I could phrase this?

Reply:
Yeah, "Any other response is interpreted as a signal that the UE is captive."